docs/Modpacks/KubeJS/Startup-Scripts/Ore-Generation.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Ore Generation
+---
+
+# Ore Generation
+
+
+## Adding a Custom Ore Vein
+
+```js title="piss_vein.js"
+GTCEuServerEvents.oreVeins(event => {
+    event.add("kubejs:piss_vein", vein => {
```

Comment:
I'd recommend changing this xD

```suggestion
```js title="custom_vein.js"
GTCEuServerEvents.oreVeins(event => {
    event.add("kubejs:custom_vein", vein => {
```

Reply:
forgot to change that when copying it over, i wasn't the one who came up with that name